src/prefect/cli/execute.py

```diff
@@ -81,10 +81,14 @@ def _execute_flow_run():
 
         with prefect.context(secrets=secrets, loading_flow=True):
             flow = storage.get_flow(storage.flows[flow_data.name])
-            environment = flow.environment
 
-            environment.setup(flow)
-            environment.execute(flow)
+        with prefect.context(secrets=secrets):
+            if getattr(flow, "run_config", None) is not None:
+                flow.run()
```

Comment:
This is both cool to see but also I don't think will work immediately; right now this call will do a few things that aren't desirable in the context of a Cloud run:
- will wait for a scheduled start time
- will attempt to execute task retries if they are configured over a long time horizon (> 10 mins)
- possibly other context manipulation?

Reply:
Switching to creating a `FlowRunner` and calling `run` on that.